### PR TITLE
[4.0] configuration.php-dist

### PR DESCRIPTION
**Pull request by brianteeman**
update to match actual generated file

- remove quotes from numbers
- change 0/1 to false/true where appropriate
- update comments

### CODE REVIEW

File: installation/configuration.php-dist [hunk 1]
```diff
@@ -26,24 +26,24 @@
...
-	public $display_offline_message = '1';
+	public $display_offline_message = 1;
 	public $offline_image = '';
 	public $sitename = 'Joomla!';            // Name of Joomla site
 	public $editor = 'tinymce';
-	public $captcha = '0';
-	public $list_limit = '20';
-	public $access = '1';
-	public $frontediting = '1';
+	public $captcha = 0;
```
Comment (Quy) [inline installation/configuration.php-dist:35]:
```suggestion
	public $captcha = '0';
```

Comment (brianteeman) [reply installation/configuration.php-dist:35]: I did not change that because it is generated by global config as '0'

Comment (Quy) [reply installation/configuration.php-dist:35]: It should be string as other possible values are string such as `public $captcha = 'recaptcha';`.

Comment (brianteeman) [reply installation/configuration.php-dist:35]: then it is a bug to be resolved elsewhere

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a44ef7c5a7037429b1bd26ad190701d7c0c4b08d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27254">issues.joomla.org/tracker/joomla-cms/27254</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on a44ef7c5a7037429b1bd26ad190701d7c0c4b08d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27254">issues.joomla.org/tracker/joomla-cms/27254</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27254">issues.joomla.org/tracker/joomla-cms/27254</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**